### DISCUSSION TITLE
Overview, Update and Questions 13-10-2013

### DISCUSSION THREAD

**Discussion by DominiqueProulx** [Broke-Brokers/soen341_fproject2023#106] [General]
13 OCTOBRE 2023

### REQUIREMENT SPECIFICATION From : https://docs.google.com/document/d/19pAZIpSTlNotPQtnnhiAsl9joSGEGWpt/edit
1.Develop and implement each feature :  update in the development section
2. Write 2 acceptance tests for the user stories identified in the first sprint.    in progress issue #78

**3.Set up your Continuous Integration pipeline in your repository**
- Question 1 : I need help understanding it . Anybody has a thought or idea on how to implement this?: 
  
> Set up your Continuous Integration pipeline in your repository.  Unit tests must run automatically for every push/pull request to the main/master branch. 

**4. Plan for the next sprint the same way that you did for Sprint 2+ user stories**
   - I will do the plan for next sprint (issue #104 created)
- Question 2 : I can do the user stories since I already started the breakdown (when I thought search for brokers and offers were in sprint2) any objections?

5. meeting minutes. in progress. due end of sprint 
6. Contribution logs. in progress. due end of sprint
7. Evaluate your peers on their Initiative and leadership during the sprint by [filling a form ](https://forms.office.com/r/5hYTKcqazM)for each of them.  Issue #105 created
8. Team demo. end of sprint 

[The rubrik is out!](https://docs.google.com/spreadsheets/d/10119rMRk2c2sC7K8f13Un3VYc4RgqH4d/edit#gid=282116742)
![image](https://github.com/Broke-Brokers/Broke_BRokers-soen341projectF2023/assets/129322102/45ff5b43-063c-41ec-864c-03264fbfe4a2)

---

### DESIGN

**Structure and page Hierarchy:** 

- https://github.com/Broke-Brokers/Broke_BRokers-soen341projectF2023/issues/51
Design hierarchy needs to be re-done with the proper file names in jsx (I am unacquainted with the sitemap that a react app creates- is it the same ? If so I can do it.) 

 **Databases:** 

Database for property are created and awesome! 
  
- Question 3: is it easy to extract a list of the fields so we can directly reference them without using dummy names? Or else I’ll take them directly from your screenshot in issue #59 😊

---

### DEVELOPMENT AND IMPLEMENTATION

- homepage is in progress
- login page is done 
- navigation is in progress
- property page is in progress

----

### ORGANIZATION 
Images and source material
- Question 5 : Should’nt the folder that was created for the images followed when main branch was moved?  I feel it should have been placed here, what do you think?: 
	  
![image](https://github.com/Broke-Brokers/Broke_BRokers-soen341projectF2023/assets/129322102/237ac4e9-7c17-45db-a509-d4d9f980e71b)

### THINGS WE NEED TO DO

•	We need to assign ourselves as verificators on issues. 
•	Write how you are testing components.

**Comment by DominiqueProulx** [comment]
My questions 

- Question 1 : I need help understanding it . Anybody has a thought or idea on how to implement this?:
- Question 2 : I can do the user stories since I already started the breakdown (when I thought search for brokers and offers were in sprint2) any objections?
- Question 3: is it easy to extract a list of the fields so we can directly reference them without using dummy names? 
- Question 4 : n/a
- Question 5 : Should’nt the folder that was created for the images followed when main branch was moved?  I feel it should have been placed here, what do you think?:

**Reply by DominiqueProulx** [reply]
question 4 is n/a - removed

**Reply by DominiqueProulx** [reply]
Questions 1 -5 answered during meeting 14-10-2023

**Comment by isaisa002** [comment]
I suggested to move broker and administrator user story to sprint 3

**Reply by DominiqueProulx** [reply]
we have to do the CRUD for both in sprint 2 per the moodle document, unfortunately :S 

![image](https://github.com/Broke-Brokers/Broke_BRokers-soen341projectF2023/assets/129322102/5efc9424-4aca-467e-8b15-fcaad378ce3d)

a) and b) is done from homebuyers, renter and brokers user stories
c) is done from brokers user story
d) is done from sysadmin user story


These features I have already removed from task breakdown of each user story and moved them to issues for sprint3 
![image](https://github.com/Broke-Brokers/Broke_BRokers-soen341projectF2023/assets/129322102/ef35989e-e443-4204-aa92-5e69def05312)
